controller/controller_server/utils.py

```diff
@@ -116,15 +239,15 @@ def validate_create_volume_request(request):
 
     logger.debug("validating secrets")
     if request.secrets:
-        validate_secret(request.secrets)
+        validate_secrets(request.secrets)
 
     logger.debug("validating storage class parameters")
     if request.parameters:
-        if config.PARAMETERS_POOL not in request.parameters:
+        if config.PARAMETERS_POOL in request.parameters:
+            if not request.parameters[config.PARAMETERS_POOL]:
+                raise ValidationException(messages.wrong_pool_passed_message)
```

Comment:
wrong, or more specifically, empty?

Reply:
no, just a more specific message

Reply:
the message is 'pool parameter must provide value for it'

Reply:
I can change the name of the message

Reply:
right, so could you change it to something like `pool parameter should not be empty` while you're at it?

Reply:
done